go/libraries/doltcore/sqle/enginetest/dolt_queries_help.go

```diff
@@ -0,0 +1,231 @@
+				Query: "select short_description from dolt_help where name='dolt_clean'",
+				Expected: []sql.Row{
+					{"Deletes untracked working tables"},
+				},
+			},
+			{
+				Query: "select short_description from dolt_help where name='dolt_checkout'",
+				Expected: []sql.Row{
+					{"Switch branches or restore working tree tables"},
+				},
+			},
+			{
+				Query: "select short_description from dolt_help where name='dolt_fetch'",
+				Expected: []sql.Row{
+					{"Download objects and refs from another repository"},
+				},
+			},
+		},
+	},
+
+	{
+		Name:        "dolt_help long descriptions are correct",
+		SetUpScript: []string{},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query: "select long_description from dolt_help where name='dolt_add'",
+				Expected: []sql.Row{
+					{`
+This command updates the list of tables using the current content found in the working root, to prepare the content staged for the next commit. It adds the current content of existing tables as a whole or remove tables that do not exist in the working root anymore.
+
+This command can be performed multiple times before a commit. It only adds the content of the specified table(s) at the time the add command is run; if you want subsequent changes included in the next commit, then you must run dolt add again to add the new content to the index.
+
+The dolt status command can be used to obtain a summary of which tables have changes that are staged for the next commit.`},
+				},
+			},
+			{
+				Query: "select long_description from dolt_help where name='dolt_cherry_pick'",
+				Expected: []sql.Row{
+					{`
+Applies the changes from an existing commit and creates a new commit from the current HEAD. This requires your working tree to be clean (no modifications from the HEAD commit).
+
+Cherry-picking merge commits or commits with table drops/renames is not currently supported. 
+
+If any data conflicts, schema conflicts, or constraint violations are detected during cherry-picking, you can use Dolt's conflict resolution features to resolve them. For more information on resolving conflicts, see: https://docs.dolthub.com/concepts/dolt/git/conflicts.
+`},
+				},
+			},
+			{
+				Query: "select long_description from dolt_help where name='dolt_revert'",
+				Expected: []sql.Row{
+					{`Removes the changes made in a commit (or series of commits) from the working set, and then automatically commits the result. This is done by way of a three-way merge. Given a specific commit (e.g. <b>HEAD~1</b>), this is similar to applying the patch from <b>HEAD~1..HEAD~2</b>, giving us a patch of what to remove to effectively remove the influence of the specified commit. If multiple commits are specified, then this process is repeated for each commit in the order specified. This requires a clean working set.
+
+Any conflicts or constraint violations caused by the merge cause the command to fail.`},
+				},
+			},
+		},
+	},
+
+	{
+		Name:        "dolt_help arguments are correct",
```

Comment:
Same comment here, loosen up these test to make them less likely to break when we change things in the future.

Reply:
What you have in the bats tests is much closer to what we want here.